modules/tutorials/pages/create-couchbase-cluster.adoc

```diff
@@ -0,0 +1,46 @@
+= Create and Deploy a Cluster with Capella Free Tier
+:description: Learn how to create and deploy a cluster using Capella Operational Free Tier.
+:page-topic-type: tutorial
+:page-pagination: full
+:page-toclevels: 2
+:navtitle: Create and Deploy a Cluster
+
+[abstract]
+{description}
+
+== Prerequisites
+
+Before starting this tutorial, you must have a Couchbase Capella account. 
+If you do not have one already, xref:cloud:get-started:create-account.adoc[Create an Account]. 
+
+== Create and Deploy a Cluster
+
+Every Capella account includes one free tier operational cluster. 
+In this tutorial, you will use the free tier cluster to create and manage student records. 
+
+To create and deploy a cluster: 
+
+. Sign in to https://cloud.couchbase.com/[Couchbase Capella]. 
+
+. On the *Operational Clusters* page, click btn:[Create Cluster].
+
+. Select *My First Project* as the project for your cluster. 
+
+. Under *Cluster Option*, select btn:[Free].
+
+. In the *Cluster Name* field, enter *student-cluster*.
+
+. (Optional) Provide a description of your cluster.
+
+. Select one of the available cloud service providers.  
+
+. Select an available geographic region for your cluster.
++
+TIP: If you are unsure which cloud provider and region to choose, select AWS and US East (N.Virginia).
```

Comment:
This makes it look like the users need to change something whereas they actually can just accept the defaults. Also, while the default region was US East for me, it wasn't N.Virginia, so I guess the precise location may change. Maybe omit that bit.

```suggestion
TIP: If you are unsure which cloud provider and region to choose, select the default options, for example *AWS* and *US East*.
```